expression/expr_to_pb.go

```diff
@@ -88,9 +88,13 @@ func (pc PbConverter) constantToPBExpr(con *Constant) *tipb.Expr {
 	var (
 		tp  tipb.ExprType
 		val []byte
-		d   = con.Value
 		ft  = con.GetType()
 	)
+	d, err := con.Eval(nil)
+	if err != nil {
+		log.Errorf("Fail to eval expression, err: %s", err.Error())
```

Comment:
s/Fail to eval expression/Fail to eval constant/

Reply:
OK, I'll change it.